spec/spec_helper.rb

```diff
@@ -63,6 +57,8 @@
   c.filter_sensitive_data('<GOOGLE_API_KEY>') { ENV['GOOGLE_API_KEY'] }
   c.filter_sensitive_data('<GOOGLE_API_CX_EN>') { ENV['GOOGLE_API_CX_EN'] }
   c.filter_sensitive_data('<GOOGLE_API_CX_CY>') { ENV['GOOGLE_API_CX_CY'] }
```

Comment:
same question. can these now be removed?

Reply:
Yes. Updated ♻️